gradle/jacoco.gradle

```diff
@@ -5,13 +5,13 @@ ext {
     ModuleCoverageThresholds = [
         "assessment"    :          ["INSTRUCTION": 0.928, "CLASS":  0],
         "athena"        :          ["INSTRUCTION": 0.872, "CLASS":  2],
-        "atlas"         :          ["INSTRUCTION": 0.906, "CLASS":  5],
+        "atlas"         :          ["INSTRUCTION": 0.905, "CLASS":  5],
         "buildagent"    :          ["INSTRUCTION": 0.793, "CLASS":  0],
         "communication" :          ["INSTRUCTION": 0.925, "CLASS":  2],
         "core"          :          ["INSTRUCTION": 0.861, "CLASS": 18],
-        "exam"          :          ["INSTRUCTION": 0.938, "CLASS":  0],
+        "exam"          :          ["INSTRUCTION": 0.937, "CLASS":  0],
         "exercise"      :          ["INSTRUCTION": 0.928, "CLASS":  0],
-        "fileupload"    :          ["INSTRUCTION": 0.927, "CLASS":  1],
+        "fileupload"    :          ["INSTRUCTION": 0.917, "CLASS":  1],
```

Comment:
Decreasing by 1% is a bit too much, pleae try to revise some tests to verify the new code